sim/druid/runes.go

```diff
@@ -109,7 +227,7 @@ func (druid *Druid) applyStarsurge() {
 		SpellSchool: core.SpellSchoolArcane,
 		ProcMask:    core.ProcMaskSpellDamage,
 		Flags:       core.SpellFlagAPL,
-
+		SpellCode:   int32(SpellCode_DruidStarfire), // Please check if this is right - Starsurge affected by all Starfire talents/procs?
```

Comment:
We wouldn't want to do this. SpellCodes are meant to be a way to identify spells without having to iterate through the list of different ranks of them. This would mean that Starsurge effectively is Starfire. I would give Starsurge its own spell code if it needs to be checked for